### DISCUSSION TITLE
Welcome to Odoo Discussions!

### DISCUSSION THREAD

**Discussion by williamvietnam** [williamvietnam/python#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JanitaRiyaz** [comment]
Hello, My name is janita. I am info tech student.  I'm a web developer and love to use python backend framework Django. I will add value in this community by sharing my experience and thoughts.

**Reply by williamvietnam** [reply]
you can study erp with me, i am building tutorials about odoo | architecture of odoo same like django
